src/clr-angular/utils/smart-popover/smart-anchor.spec.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ *
+ */
+
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { ClrSmartPopoverAnchor, ClrSmartPopoverEventsService, ClrSmartPopoverToggleService } from '@clr/angular';
+import { SMART_POPOVER_SERVICES } from './smart-popover.module';
+import { spec, TestContext } from '../testing/helpers.spec';
```

Comment:
Did we decide to try and expand the use of these utilities or were we going to change them?

Reply:
I've been using them where it was easy to drop them in. I haven't modified them and would prefer not to for this PR.